ast/src/parsed/mod.rs

```diff
@@ -694,6 +726,7 @@ pub enum FunctionDefinition {
     /// Array expression.
     Array(ArrayExpression),
     /// Prover query. The Expression usually is a LambdaExpression.
+    /// TODO can we replace this by a query-marked lambda expcession?
```

Comment:
Still want to see if this makes sense.

Reply:
still no vscode spell checker? :D

Reply:
It's removed code!

Reply:
I mean `expcession`

Reply:
I know, but I was able to replace it and deleted the comment.